docs/source/en/package_reference/environment_variables.md

```diff
@@ -74,6 +74,12 @@ small files will be duplicated to ease user experience while bigger files are sy
 
 For more details, see the [download guide](../guides/download#download-files-to-local-folder).
 
+### HF_HUB_KEEP_LOCK_FILES
+
+Lock files are used to ensure the downloading in parallel won't overwrite. By default, the lock file will be deleted
+automatically. However, the deletion would cause concurrency issues in rare cases. Set this to `True` to prevent this
+kind of concurrency issue.
```

Comment:
```suggestion
Lock files are used to avoid concurrency issues when downloading files in parallel, especially to ensure a single file is not downloaded multiple times.
By default, the lock file will be deleted automatically to keep the cache system clean. However, this deletion can cause concurrency issues in rare cases.
Set `HF_HUB_KEEP_LOCK_FILES=1` to prevent this kind of concurrency issue.
```